test/memory/sungeun/refCount/domainMaps.lm-numa.good

```diff
@@ -77,4 +77,4 @@ Create domain and array:
 	0 bytes leaked
 	0 private objects leaked
 total:
-	1104 bytes leaked
+	0 bytes leaked
```

Comment:
These deltas make me wonder: Can we simply retire all of the special domainMaps*.good files other than the "standard" one now?